.github/ISSUE_TEMPLATE/1-feature-request.yml

```diff
@@ -0,0 +1,66 @@
+name: Feature request
+description: Submit a new idea
+title: "[Idea]: "
+labels: ["enhancement"]
+body:
+- type: textarea
+  id: what
+  attributes:
+    label: What is the problem you are trying to solve?
+    placeholder: A clear and concise description of what the problem is.
+  validations:
+    required: true
+
+- type: textarea
+  id: how
+  attributes:
+    label: Which solution do you envision (roughly)?
+    placeholder: Once the work is started, a design doc would be great!
+  validations:
+    required: true
+
+- type: textarea
+  id: alternatives
+  attributes:
+    label: Have you considered any alternatives?
+    placeholder: A clear and concise description of any alternative solutions or features you've considered.
+  validations:
+    required: false
+
+- type: textarea
+  id: context
+  attributes:
+    label: Any additional context to share?
+    placeholder: Add any other context or screenshots about the feature request here.
+  validations:
+    required: false
+
+- type: dropdown
```

Comment:
Does it make sense to request this from community members? I'm not sure most will be able to accurately estimate the effort involved.

Reply:
This dropdown defaults to "Not sure" which let community members not have to think about it.